### PR TITLE
DAOS-5723 Automate IOR hard on Frontera for weekly run

### PR DESCRIPTION
**Pull request by berserk-fury**
Automating 2 ior hard testcases:
- scaling server2client 1to4
- constant client count at 16, scaling server

Codebase enhancements
- Run python linter on run_testlist.py file
- Update get_ior_results.sh file to collec ior results
  into a csv file
- Add a function to wait for all the DAOS servers to
  be up or the function timeout

Signed-off-by: Martinez Montes, Jonathan <jonathan.martinez.montes@intel.com>

### CODE REVIEW

**Comment by berserk-fury** [comment]
I need to adjust some timeouts

File: run_build.sh [hunk 1]
```diff
@@ -2,9 +2,42 @@
...
+    CURRENT_FILE=${LATEST_DAOS}/${i}
+
+    if [ ! -f ${CURRENT_FILE} ]; then
+      echo "Error: missing file ${CURRENT_FILE}"
+      exit 1
+    fi
+
+    if ! ldd ${CURRENT_FILE} | grep -q daos ; then
+      echo "Error: file ${CURRENT_FILE} does not link DAOS libraries"
+      exit 1
```
Comment (mjean308) [inline run_build.sh:36]: same comment as in PR-26

**Comment by berserk-fury** [comment]
Unfortunately, all my three 24, 25, and 26 pull requests depends on each other. GitHub does not show the incremental change of each PR but the whole diff against master.
Fortunately, they are like Matryoshka Dolls. So, I recommend you that the review goes on this order 24, 25 and finally 26.
Sorry for the inconvenience.

**Review by mjean308 — APPROVED** [review]
lgtm

**Review by sylviachanoiyee — APPROVED** [review]

**Pull request merged**